mod-configuration-server/src/test/java/org/folio/rest/RestVerticleTest.java

```diff
@@ -1395,45 +1356,44 @@ public void checkURLs(TestContext context) {
 
   private void createSampleRecords(TestContext context) {
     try {
-      //save config entry
-      String sample = getFile("kv_configuration.sample");
-
-      ConfigurationRecordBuilder baselineFromSample = ConfigurationRecordBuilder.from(sample);
-
+      JsonObject configRecord = new ConfigurationRecordBuilder()
+        .withModuleName("DUMMY")
+        .withDescription("dummy module for testing")
+        .withConfigName("dummy_rules")
+        .withCode("config_data")
+        .withValue("")
+        .create();
+
+      configRecord.put("default", "true");
+
+      ConfigurationRecordBuilder baselineFromSample = ConfigurationRecordBuilder.from(configRecord);
```

Comment:
Why does the code need to reinterpret a config record generated using the builder back through the builder?

Reply:
because the rest of the test needs a ConfigurationRecordBuilder object representing baseline record data that it can use to mutate and build variant records from.  The only way I saw to get such a thing is to use configurationObjectBuilders "with()" method.  "with()" can take either a string or a JsonObject representing the record and return a builder object you can use to make variants.  Originally, this test loaded string data from a file directly into "with()" to get such an object.  But we removed that data.  Java doesn't deal well with inline multiline strings, so I really did not want to try to construct a string representation of a record in the code itself.  That leaves generating from a JsonObject, which ConfigurationRecordBuilder Itself can create.  So, I solved the problem by using ConfigurationRecordBuilder's constructor, which returns a jsonObject representation of the record.  That also allows me to set the "default" value manually on the object.  Then I just fed the JsonObject back into the "with()" method to get the builder object I needed.

Reply:
I'm a little surprised that works as intended, because the `from` method for the `ConfigurationRecordBuilder` appears to disregard any properties of the JSON it does not understand. 

I imagine this means that the eventual configuration record from the builder does not include the `default` property.

Reply:
I'm glad you decided to steer clear of using a string representation of the record. I agree those are brittle and difficult to follow.

Reply:
Actually, default does get set in the created record-I know because I had the tests dumping data to console while I was trying to adjust all the totals in the .csv file to match query results, and I could see the field.  I believe one of the test urls in the CSV file even looks for it, which is why I went to the trouble of making sure it was set.  Only one of the records had the field set-which I imagine is because if you have multiple config records with the same name, only one of them can be set as the default-which I expect would be the last one created with the property set, and that's the behavior I saw.  Only the last created record had the field set.

Reply:
Yeah, here you go-there are several urls using it as a search parameter-they would fail if it wasn't getting set at all:

GET , http://localhost:<port>/configurations/entries?query=default==true&facets=code&facets=module , get list of configuration tables , 200 , 1
GET , http://localhost:<port>/configurations/entries?query=default==true , get list of configuration tables , 200 , 1
GET , http://localhost:<port>/configurations/entries?query=default==true%20sortBy%20code/sort.descending , get list of configuration tables , 200 , 1

Reply:
Ok, my quick inspection of the code suggested that wouldn't work. I'm often wrong.

Reply:
Well, as it turns out, you're kinda right.  When I built withDefault into the constructor and started using that, the URLs above retrieve all four records-all of them have default set to true.  So I have no idea why only one of them was showing the setting before.